src/cli.ts

```diff
@@ -296,44 +320,46 @@ export type ICallOrInvokeProps = ICallOrInvokeProps_ &
 
 program
   .command('invoke <file>')
-  .requiredOption('--address <address>', 'Address of contract to invoke.')
-  .requiredOption('--function <function>', 'Function to invoke.')
+  .description('Invoke a function on a warped contract using the Solidity abi')
```

Comment:
As there is an option  `--use-cairo-abi` it should be `Uses Solidity abi by default`

Reply:
I don't think we should be advertising the --use-cairo-abi to the user; its really for debugging purposes.
If somehow we can remove it from it a production build that would be ideal IMO